starter-code/scripts/articleView.js

```diff
@@ -37,11 +37,15 @@ articleView.handleAuthorFilter = function() {
     //         defining. "$(this)" is using jQuery to select that element, so we can chain jQuery methods
     //         onto it.
     if ($(this).val()) {
+      $('.article-body').hide();
+      console.log($(this).val());
+      $('article[data-author="' + $(this).val() +'"] .article-body').fadeIn(1000);
       // TODO: If the select box was changed to an option that has a value, we need to hide all the articles,
       //       and then show just the ones that match for the author that was selected.
       //       Use an "attribute selector" to find those articles, and fade them in for the reader.
 
     } else {
+      $('.article-body').show();
```

Comment:
this should be `$('article').not('.template').show();` because with the change above, the `.article-body` won't be visible if the `<article>` is still hidden.

Reply:
then in line 42, you wouldn't need to include `.article-body`